Apps/WebAPI/sdLitica.WebAPI/appsettings.Development.json

```diff
@@ -16,5 +16,12 @@
   "TimeSeries": {
     "InfluxHostName": "http://localhost:8086",
     "InfluxDatabase": "sdLitica"
+  },
+  "Messages": {
```

Comment:
If this is a config map containing details about connectivity with RbbitMQ, maybe it will be better calling it RabbitMQConfig? Messages is a bit misleading.

Reply:
Agree. I will change it.